src/metabase/lib/field.cljc

```diff
@@ -283,14 +284,35 @@
   [query stage-number field-ref]
   (lib.temporal-bucket/available-temporal-buckets query stage-number (resolve-field-metadata query stage-number field-ref)))
 
+(defn- fingerprint-based-default [fingerprint]
```

Comment:
nit: "fingerprint-based default" *what*? This doesn't say what it returns. Suggest `fingerprint-based-default-unit`.

Reply:
Will fix in a different PR.